core/domain/story_domain_test.py

```diff
@@ -625,6 +641,19 @@ def test_add_node_validation(self) -> None:
             'next node id for the story'):
             self.story.add_node('node_4', 'Title 4')
 
+    def test_delete_node_validation(self) -> None:
```

Comment:
This test name still needs to be fixed as I commented on here: https://github.com/oppia/oppia/pull/16681#discussion_r1045160043

Reply:
done!